CHANGES.md

```diff
@@ -21,6 +21,10 @@ ones in. -->
 [#463](https://github.com/cylc/cylc-uiserver/pull/463) - Fixed failure to
 connect to workflows when they were restarted.
 
+[#455](https://github.com/cylc/cylc-uiserver/pull/455)- Added a specific
+warning for cases where workflow needs upgrade, and an upgrade toggle in
+cylc play dialog.
+
```

Comment:
If this is going in 1.4.0 then will need to move

Reply:
Have done that, but converted to draft - will un-draft after  1.3.0 released.